docs/source/conf.py

```diff
@@ -15,8 +15,11 @@
 import os
 import sys
 import re
+
 sys.path.insert(0, os.path.abspath('../../Wrappers/Python/'))

```

Comment:
why do you need this?

Reply:
it tells sphinx where to look for docstrings I think?